pkg/chain/local/local.go

```diff
@@ -470,6 +473,22 @@ func (c *localChain) ReportRelayEntryTimeout() error {
 	return nil
 }
 
+func (c *localChain) IsEntryInProgress() (bool, error) {
+	return c.currentEntryStartBlock.Uint64() != 0, nil
+}
+
+func (c *localChain) CurrentEntryStartBlock() (*big.Int, error) {
+	return c.currentEntryStartBlock, nil
+}
+
+func (c *localChain) CurrentRequestPreviousEntry() ([]byte, error) {
+	return c.currentRequestPreviousEntry, nil
+}
+
+func (c *localChain) GroupPublicKey() ([]byte, error) {
+	return c.groups[0].groupPublicKey, nil
+}
+
```

Comment:
Honestly, I think it's better if we `panic("not implemented")` for them, we can save some time in the future while figuring out why unit test does not work as expected.

And if we do not decide for panic we should assign `currentRequestPreviousEntry` in `SubmitRelayEntry` and `currentEntryStartBlock` somewhere else - hard to say where.

Reply:
sure, replaced with `panic("not implemented")`